data/genomics/homo_sapiens/illumina/10xgenomics/cellranger/10k_pbmc_cmo/10k_pbmc_cmo_config.csv

```diff
@@ -0,0 +1,18 @@
+[gene-expression]
+reference,/references/deleteme
```

Comment:
is this path with deleteme intentional?

Reply:
oops, that's an artifact from when I tested the subsampled data before creating the PR

`deleteme` is simply the name of the reference, I'll change it to something more realistic

Reply:
i might have seen it in some other file as well

Reply:
you definitely did! it was in two of the `cellranger multi` config files

very good eye 👌🏼

Reply:
`deleteme` is now deleted 👍🏼